docs/path-resolution.rst

```diff
@@ -313,6 +315,46 @@ interpreted as absolute paths on disk.
 If the base path itself is relative, it is also interpreted as relative to the current working
 directory of the compiler.
 
+.. _base-path-normalization-and-stripping:
+
+Base Path Normalization and Stripping
+-------------------------------------
+
+When source file paths are specified on the command line, the base path affects the source unit
+names assigned to them in the compiler's VFS.
+To compute the names, both base path and source file paths must first be converted to a canonical form.
+This ensures that the result is predictable and as platform-independent as possible:
+
+- If a path is relative, it is made absolute by prepending the current working directory to it.
+
+  - If the path to the working directory contains symbolic links, they are resolved into actual
```

Comment:
Is this indention correct?

Reply:
Yes. That's a nested bullet point. The parent point speaks about "working directory" and this extra point explains a related detail - it says what happens if the path to the working directory has symbolic links in it. I could make it a separate point but it's only really relevant in the context of relative paths.

And if you mean specifically the RST syntax, the indentation is always long enough to match the bullet point of the parent. So here it's two spaces. If the parent was `1. If a path ...` then you'd need to indent with 3 spaces. With `123. If a path ...` you'd need 5 and so on.